resp/responses/configurations/shortperiod.yaml

```diff
@@ -144,6 +144,18 @@ response:
 
   Nanometrics Dataloggers Connected to Short Period Sensors:
     sensors:
+    - sensors:
+      - L4C
```

Comment:
Unless we plan on plugging a 1D sensor into a centaur, I'd not bother with this section.

Reply:
Ah, just spotted @salichon bet me too the punch!

Reply:
@salichon @ozym consider it gone!